src/main/java/org/broadinstitute/consent/http/db/LibraryCardDAO.java

```diff
@@ -66,16 +67,22 @@ void updateLibraryCardById(@Bind("libraryCardId") Integer libraryCardId,
       " institution.create_user AS i_create_user_id, " +
       " institution.create_date AS i_create_date, " +
       " institution.update_date AS i_update_date, " +
-      " institution.update_user AS i_update_user_id " +
+      " institution.update_user AS i_update_user_id, " +
+      " ld.daa_id " +
       " FROM library_card AS lc " +
       " LEFT JOIN institution " +
       " ON lc.institution_id = institution.institution_id" +
+      " LEFT JOIN lc_daa ld ON lc.id = ld.lc_id " +
       " WHERE lc.user_id = :userId")
   List<LibraryCard> findLibraryCardsByUserId(@Bind("userId") Integer userId);
 
   @RegisterBeanMapper(value = LibraryCard.class)
   @UseRowReducer(LibraryCardReducer.class)
-  @SqlQuery("SELECT * FROM library_Card WHERE institution_id = :institutionId")
+  @SqlQuery("SELECT library_Card.*, " +
+      "ld.daa_id " +
+      "FROM library_Card " +
+      "LEFT JOIN lc_daa ld ON library_Card.id = ld.lc_id " +
+      "WHERE library_Card.institution_id = :institutionId")
```

Comment:
We should make these text blocks instead of concatenated strings. Semgrep sometimes complains about this, and text blocks are easier to read, copy, paste, etc.